tests/core/test_actions.py

```diff
@@ -2665,3 +2664,51 @@ async def test_action_extract_slots_non_required_form_slot_with_from_entity_mapp
         domain,
     )
     assert events == [SlotSet("form1_info1", "info1"), SlotSet("form1_slot1", "Filled")]
+
+
+async def test_action_extract_slots_returns_slot_set_even_if_slot_value_is_unchanged():
+    event_with_slot_entity = UserUttered(
+        text="I am a text",
+        intent={"name": "intent_with_entity"},
+        entities=[{"entity": "entity", "value": "value"}],
```

Comment:
suggest to set the name & value as variables so there's no chance of mismatch lower down in the test